.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,27 +6,39 @@ labels: bug
 
 ---
 
+**⚠️IMPORTANT⚠️ if you do not fill this out, we cannot help you.**
+
 **Describe the bug**
 A clear and concise description of what the bug is.
 
+**Plaintext**
+**⚠️IMPORTANT⚠️ The below code is non-negotiable for "Ciphey didn't decrypt...." problems. If you do not tell us your plainttext, we will not help you.** 
+
+```
+Include your plaintext here, replacing this 
+```
+
+**Verbose Output**
+**⚠️IMPORTANT⚠️ Verbose outputt will tell us why it's not working the way we expected it to be.**
+Run Ciphey with `ciphey -vvv` and paste the results (either as a file, or copied & pasted) into the GitHub issue comment. 
+
+**Versions:**
+**⚠️IMPORTANT⚠️ We need this information because Ciphey does not work effectively on all systems.**
```

Comment:
Could you soften this (i.e. "Different environments will induce different bugs in Ciphey")